R/utils.R

```diff
@@ -200,3 +200,47 @@ ensure_data.table <- function(data) {
   }
   return(data)
 }
+
+#' @title Print information about a `forecast` object.
```

Comment:
Suggest making this title case for consistency with (most...) other functions

```suggestion
#' @title Print Information About A Forecast Object
```